content/applications/inventory_and_mrp/inventory/product_management/configure/type.rst

```diff
@@ -2,230 +2,346 @@
-  replaceable and essential, but exact counts are unnecessary; or
-- :guilabel:`Service` for sellable service products that are performed, and not tracked with stock
-  counts (i.e. maintenance, installation, or repair services).
+- :guilabel:`Goods`: A tangible, material object (e.g. anything from a hamburger to a house).
 
-  .. image:: type/product-form.png
-    :align: center
-    :alt: Set a product type on the product form.
+- :guilabel:`Service`: An intangible, immaterial offering (e.g. a repair, a haircut, call center
+  assistance, etc.
+
+- :guilabel:`Combo`: Any mix of goods or services (e.g. a new car (*good*) with an oil change
+  included (*service*)).
 
 .. note::
-   The product types listed above are part of the standard *Inventory* app. For access to the fields
-   below, :ref:`install <general/install>` the corresponding apps **in addition** to *Inventory*.
-
-- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Appointments* app.
-  Requires the installation of the *Calendar* app and *Pay to Book* (`appointment_account_payment`)
-  module
-- :guilabel:`Combo`: create discounted products sold in a bundle. Requires the installation of the
-  *PoS* app.
-- :guilabel:`Event Ticket`: sold to attendees wanting to go to an event. Requires the installation
-  of the *Events* app
-- :guilabel:`Event Booth`: sold to partners or sponsors to set up a booth at an event. Requires the
-  installation of the *Events* app
-- :guilabel:`Course`: sell access to an educational course. Requires the installation of the
-  *eLearning* app
-
-Compare types
-=============
-
-Below is a summary of how each product type affects common *Inventory* operations, like transfers,
-reordering rules, and the forecasted report. Click the chart item with an asterisk (*) to navigate
-to detailed sections.
+  Due to their immaterial nature, services are not trackable in Odoo's *Inventory* application.
+
+.. _inventory/product_management/manufacture:
+
+Configuring goods
+=================
+
+Selecting :guilabel:`Goods` as the product type automatically triggers the appearance of a few
+fields in Odoo:
+
+- :guilabel:`Inventory` tab: From here,
+  :doc:`purchasing and manufacturing routes <../../shipping_receiving/daily_operations/use_routes>`
+  and product logistics (such as product weight and customer lead time) can be specified.
+
+- :ref:`Invoicing Policy <inventory/product_management/invoicing-policy>` field.
+
+.. note::
+   The :guilabel:`Invoicing Policy` field will only appear if the *Sales* app is installed.
+
+- :ref:`Track Inventory <inventory/product_management/tracking-inventory>` field.
+
+- Smart buttons: Some smart buttons are triggered right away when :guilabel:`Goods` is selected;
+  others show upon additionally selecting a :guilabel:`Track Inventory` method. For example,
+  :guilabel:`On Hand` and :guilabel:`Forecasted` will display when :guilabel:`Track Inventory` is
+  ticked.
```

Comment:
I see why you say this and would agree, but I enumerate the most important smart buttons (for inventory, since this is an inventory doc) later in this doc. So instead, I am adding the following to point users in the right direction: "In general, the smart buttons on a product form help execute :ref:`inventory operations
  <_inventory/product_management/inventory-ops-by-product-type>.`"